shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/executor/UserExecutorGroup.java

```diff
@@ -46,4 +48,21 @@ private UserExecutorGroup() {
     public static UserExecutorGroup getInstance() {
         return INSTANCE;
     }
+    
+    /**
+     * Close executor service.
+     */
+    @Override
+    public void close() {
+        if (null != executorService) {
+            try {
+                executorService.shutdown();
+                while (!executorService.awaitTermination(5, TimeUnit.SECONDS)) {
+                    executorService.shutdownNow();
+                }
+            } catch (final InterruptedException ex) {
+                Thread.currentThread().interrupt();
+            }
+        }
```

Comment:
UserExecutorGroup is for proxy only, spring boot maybe not need change

Reply:
ok, revert it.